packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -131,6 +131,10 @@ class BottomNavigationBar extends StatefulWidget {
   /// See [BottomNavigationBarItem.icon] for more information.
   final double iconSize;
 
+  /// Whether to show the labels of bottom navigation bar items that arent'
+  /// selected.
+  final bool hideLabels = false;
```

Comment:
Seems like this isn't used? It's also odd that it's final and always false :)

Reply:
Meant to remove this